spec/content_tagger/linking_related_content_spec.rb

```diff
@@ -0,0 +1,47 @@
+feature "Adding related content to Publisher content", new: true, content_tagger: true, frontend: true, publisher: true do
+  include ContentTaggerHelpers
+  include PublisherHelpers
+
+  let(:guide_title) { "Transaction with related content " + SecureRandom.uuid }
+  let(:guide_slug) { "transaction-with-related-content-" + SecureRandom.uuid }
+  let(:related_content_title) { "Related transaction " + SecureRandom.uuid }
+  let(:related_content_slug) { "related-transaction-" + SecureRandom.uuid }
+
+  scenario "Adding a taxon to a publisher guide" do
+    given_there_is_a_published_guide
+    when_i_add_related_content_to_the_guide
+    then_the_related_content_is_linked_to_from_the_guide
+  end
+
+  def given_there_is_a_published_guide
+    @guide_url = create_and_publish_guide(slug: guide_slug, title: guide_title)
+  end
+
+  def when_i_add_related_content_to_the_guide
+    @related_content_url = create_and_publish_guide(slug: related_content_slug, title: related_content_title)
+    visit_tag_external_content_page(slug: guide_slug)
+
+    find("input.new-base-path").set("/" + related_content_slug)
+    click_button "Add related item"
+    expect(page).to have_text(related_content_slug)
+    click_button "Update tagging"
+    expect(page).to have_text("Tags have been updated!")
+  end
+
+  def then_the_related_content_is_linked_to_from_the_guide
+    reload_url_until_match(@guide_url, :has_text?, related_content_title)
```

Comment:
should we have status codes in here? I guess it starts as 404?

Reply:
Good catch.  I'd forgotten we need a double check.